common/src/main/java/com/microsoft/identity/common/internal/request/SdkType.java

```diff
@@ -22,10 +22,24 @@
 // THE SOFTWARE.
 package com.microsoft.identity.common.internal.request;
 
+import com.microsoft.identity.common.adal.internal.AuthenticationConstants;
+
 /**
  * Enum to indicate if the request came from an app hosting Adal or Msal Sdk
  */
 public enum SdkType {
     ADAL,
-    MSAL
+    MSAL,
+    MSALCPP;
+
+
+
+    public String getProductName() {
+        if ((SdkType.ADAL == this) || (SdkType.MSAL == this)) {
+            return AuthenticationConstants.SdkPlatformFields.PRODUCT_NAME_MSAL;
+        } else {
+            return AuthenticationConstants.SdkPlatformFields.PRODUCT_NAME_MSAL_CPP;
+        }
+
+    }
 }
```

Comment:
Add new line at end of file

Reply:
798ca9f7d8e3a969707905150533c290ff12eb97